packages/react/src/spinner/spinner.tsx

```diff
@@ -0,0 +1,53 @@
+import { forwardRef, useEffect, useState } from "react";
+import { clsx } from "@postenbring/hedwig-css/typed-classname";
+
+interface SpinnerPropsInner extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The size of the Spinner
+   *
+   * @default "medium"
+   */
+  size?: "small" | "medium" | "large";
+
+  /**
+   * The tooltip title
+   *
+   * @default ""
+   */
+  title?: string;
+
+  /**
+   * The delay before showing the spinner
+   *
+   * @default 1000
+   */
+  delay?: number;
+}
+
+export type SpinnerProps = SpinnerPropsInner;
+
+/**
+ * Use spinner loading states as placeholder for your content while waiting for data to load.
+ */
+export const Spinner = forwardRef<HTMLDivElement, SpinnerProps>(
+  ({ size = "medium", title = "", delay = 1000, className, ...rest }, ref) => {
+    const [shownAfterOneMinute, setShownAfterOneMinute] = useState<boolean>(false);
+    useEffect(() => {
+      const timeout = setTimeout(() => {
+        setShownAfterOneMinute(true);
+      }, delay);
+      return () => {
+        clearTimeout(timeout);
+      };
+    });
+    return shownAfterOneMinute ? (
+      <div
+        title={title}
+        className={clsx("hds-spinner", `hds-spinner--${size}`, className as undefined)}
+        ref={ref}
+        {...rest}
+      />
+    ) : null;
```

Comment:
🔧 Jeg tror det er bedre å skjule spinneren med `visibility:hidden` i stenden for å ikke rendre den. Da ungår man layout shift når spinneren kommer til syne

Reply:
> 🔧 Jeg tror det er bedre å skjule spinneren med `visibility:hidden` eller `opacity: 0` i steden for å ikke rendre den. Da ungår man layout shift når spinneren kommer til syne.
> 
> Forsåvidt, hvis får den til fade in får vi dette "gratis" 😎

@juliuxu Jeg har nå lagt på fade in med 2s delay. Jeg fjernet prop'en for delay, for jeg tenkte at det kanskje ikke er noe så mange trenger å endre på, men det kan man jo vurdere?